### PR TITLE
Rework Main Section of Actor Sheet

### PR DESCRIPTION
**Pull request by moliloo**
Styling Main Section for a better of the sheet and a better use, the gap between the columns is intentional because of the width of the sheet, in future merges, reworking other partials, i want, in the future, to reduce the width of the Actor Sheet.

### CODE REVIEW

**Comment by Stexinator** [comment]
Can you change your editor to preserve the current linting?

**Comment by moliloo** [comment]
I'm sorry, i forget about it, beside this, do i need change other thing in my editor?

**Comment by Stexinator** [comment]
This was automatically closed due to the hotfix. Can you reopen it?

File: module/actor/witcherActor.js [hunk 1]
```diff
@@ -1,608 +1,609 @@
...
+
+			"system.coreStats.woundTreshold.current": Math.floor((baseMax + wtTotalModifiers) / wtDivider),
+			"system.coreStats.woundTreshold.max": baseMax,
+
+			"system.coreStats.stun.totalModifiers": stunTotalModifiers,
+			"system.coreStats.enc.totalModifiers": encTotalModifiers,
+			"system.coreStats.run.totalModifiers": runTotalModifiers,
+			"system.coreStats.leap.totalModifiers": leapTotalModifiers,
+			"system.coreStats.rec.totalModifiers": recTotalModifiers,
+			"system.coreStats.woundTreshold.totalModifiers": wtTotalModifiers,
```
Comment (Stexinator) [inline module/actor/witcherActor.js:202]: These needs to be sorted where the other fields are set => current and max

**Pull request closed without merging**